source/data/user_access.rst

```diff
@@ -3,7 +3,7 @@
 User Access to iRODS
 ====================
 
-The Tier-1 Data Service is currently in a pilot phase, so access to the system is strictly by invitation. If you have a use case that combines data and computing workflows and you are interested on testing the Tier-1 Data service you can contact us to discuss a possible collaboration by e-mail at the address: data@vscentrum.be
+The Tier-1 Data Service at KU Leuven is currently in a pilot phase, so access to the system is strictly by invitation. If you have a use case that combines data and computing workflows and you are interested on testing the Tier-1 Data service you can contact us to discuss a possible collaboration by e-mail at the address: data@vscentrum.be
```

Comment:
Would not explicitly say at KU Leuven personally, but it's not a dealbreaker either. Sorry for the late review, missed this message.